### ISSUE TITLE
Please could you make it clearer what in SharePoint and OneDrive is created when working with Loop workspaces and components

### ISSUE THREAD

**Issue by Darrelo365** [MicrosoftDocs/OfficeDocs-SharePoint#3737]
Hi, I'm doing some testing and trying to work out what exactly creating a workspace or a page does and where it is stored from an admin point of view.  I created a workspace and cannot see it in AAD Groups or in the SharePoint admin centre or as a folder in OneDrive.

If I make this available to my userbase of staff and students I have no feel for how to know what is being created where and anticipation of the usual SharePoint-like questions of I cannot access something, I cannot find a deleted thing, or I'm confused by permissions and have I shared it too openly, what will happen when I leave the organisation?

Thanks, Darrel

---
#### Document details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: c41c35b5-e159-ed6d-a9ff-bea0ef7f0736
* Version Independent ID: 2eca8e70-2e83-b100-890b-03b4377186e9
* Content: [Manage Loop experiences (Loop app and Loop components) in SharePoint - SharePoint in Microsoft 365](https://learn.microsoft.com/en-gb/sharepoint/manage-loop-components)
* Content Source: [SharePoint/SharePointOnline/manage-loop-components.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/public/SharePoint/SharePointOnline/manage-loop-components.md)
* Service: **sharepoint-online**
* GitHub Login: @MikePlumleyMSFT
* Microsoft Alias: **mikeplum**

**Comment by dancontoso** [comment]
We are using SharePoint storage platform, so all the benefits of SharePoint’s hard work to make disaster recovery, multi-geo, compliance, security, encryption, etc., but it’s not an end-user visible storage location like a SharePoint site or their OneDrive for Business. It’s like a site but all the content is managed by the app. A lot of developers have been wishing for this sort of SharePoint storage for a long time. Tony Chan is probably a better person to help us figure out all the right documentation to point to for all the many admin and management related questions that I’m sure will come up over time.

**Comment by dancontoso** [comment]
@scanum can you figure out how to assign this to Tony? His MS alias is in the top of the .md

**Comment by Darrelo365** [comment]
Thanks Dan. Stuff like this is confusing. As the person who created the Workspace and page I get access denied to it via the Office.com portal and the path isn't something as a SharePoint admin I'd recognise as anything in our org.

![image](https://user-images.githubusercontent.com/63899364/227910380-5d4ae2c7-7203-4365-a08f-5b5e7b918701.png)

**Comment by dancontoso** [comment]
thanks for the screenshots!

**Comment by tonysmit** [comment]
I don’t own this content. Please assign to someone else on the team. Thx



Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Comment by Darrelo365** [comment]
Hi, I'm happy to read chat elsewhere if someone can direct me to where SharePoint Admin are asking similar questions. Thanks

**Comment by dancontoso** [comment]
Hi Darrel! We're working on adding IT admin FAQ to either (a) a new article on support.microsoft.com, or (b) to an existing one. Specifically, some of the questions you're asking around storage and how admins should develop a mental model of management - is it similar to anything they're already doing or do they have to learn a bunch of new things, etc. (hint: it's similar to something you're already doing on SharePoint, since we're using SharePoint storage platform, something that we also recently blogged about, search for Zach Rosenfield SharePoint storage platform). It's taking us a little longer to pull everything together since we've built a lot of new platform capabilities for the Loop app, so apologies that it feels like we're ghosting you on this comment. I can assure you we are not, lots of activity behind the scenes here to pull together the right content here. And, safe to say that we will not detail everything in a GitHub Issue comment chain, it will be in publicly visible article/documentation. And Darrel, thanks for asking again! Now I can finally mark responding to the GitHub issue as completed and keep working on publishing the content we're putting together. ;)
